src/debug/debug-pal/unix/twowaypipe.cpp

```diff
@@ -117,8 +117,8 @@ int TwoWayPipe::Read(void *buffer, DWORD bufferSize)
     while ((bytesRead = (int)read(m_inboundPipe, buffer, cb)) > 0)
     {
         totalBytesRead += bytesRead;
-        _ASSERTE(totalBytesRead <= bufferSize);
```

Comment:
I think it would be easier to convert totalBytesRead and bytesRead variable types to DWORD

Reply:
never mind, I see that read returns an integer.